apps/web-connect/src/features/checkout/Checkout.tsx

```diff
@@ -46,9 +46,9 @@ export function Checkout() {
     }
 
 	useEffect(() => {
-		if (!stakingTabOpened && mintWithEth.isSuccess || mintWithXai.isSuccess) {	
+		if (!stakingTabOpened && (mintWithEth.isSuccess || mintWithXai.isSuccess)) {	
             setStakingTabOpened(true);
-			window.open("https://app.xai.games/staking?mint=true", '_blank');
+			window.open("https://develop.app.xai.games/staking?showKeys=true", '_blank');
```

Comment:
This needs to be app.xai.games for production. Only for building on testnet we want to have develop.app.xai.games

Reply:
This has been corrected.